### ISSUE TITLE
Add a Energyometer with the same design of speedometer and stressometer

### ISSUE THREAD

**Issue by SrRapero720** [mrh0/createaddition#408]
**WHY**
Sometimes using a visual and fast reference to get how much energy is produced (have energy or not)
Also can be util with other FE mods

**how be works**
Keeping the same design of other thing-ometers, the base of block will be the receptor of energy. of course no having shaft

![image](https://user-images.githubusercontent.com/29090346/206254632-55f74f35-6f50-41a1-adc3-e01853708771.png)

**Comment by Ganpan14O** [comment]
Based on the way I believe this mod to work I'm not sure if the idea you presented would work, however if it had a wire connector on each side and measured throughput then that could work.

**Comment by SrRapero720** [comment]
![2022-12-23_02h22_02](https://user-images.githubusercontent.com/29090346/209300020-49842868-b84b-49e9-8d0a-44deb794103d.png)

**Comment by mrh0** [comment]
> Based on the way I believe this mod to work I'm not sure if the idea you presented would work, however if it had a wire connector on each side and measured throughput then that could work.

Yes, this idea has been suggested before and the reason it isnt in the mod yet is this; and the fact that you can already see the total network usage with a pair of goggles. But i still might add it at some point.

**Comment by samuelh2005** [comment]
Similar to #90
